vic/drivers/shared_all/src/print_library_shared.c

```diff
@@ -438,18 +446,18 @@ print_layer_data(layer_data_struct *ldata,
     size_t i;
 
     fprintf(LOG_DEST, "layer_data:\n");
-    fprintf(LOG_DEST, "\tCs   : %.4f\n", ldata->Cs);
-    fprintf(LOG_DEST, "\tT    : %.4f\n", ldata->T);
-    fprintf(LOG_DEST, "\tevap : %.4f\n", ldata->evap);
+    fprintf(LOG_DEST, "\tCs   : %f\n", ldata->Cs);
+    fprintf(LOG_DEST, "\tT    : %f\n", ldata->T);
+    fprintf(LOG_DEST, "\tevap (flux): %f\n", ldata->evap);
```

Comment:
Hmm I added this because `evap` is actually a flux, but it got mixed in the state section in the layer structure. Will reorder the structure to make it clearer.